path/test/test-path-basename.ts

```diff
@@ -0,0 +1,70 @@
+import { test, assertEqual } from 'https://deno.land/x/testing/testing.ts'
```

Comment:
Please rename this file to //path/basename_test.ts
And please add it to //test.ts